dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -173,4 +173,4 @@ getRealPathReturnsQualifiedPath.name=Always return a non-null path from getRealP
 getRealPathReturnsQualifiedPath.desc=Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false.
 
 stopAppStartUponListenerException.name=Stop application for unhandled listener exception
-stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners.
+stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true.
```

Comment:
`Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true.`

This is a little confusing because it tells you to do something, but then tells you that you don't have to. It might be better to say

 `When this property is set to true, the application stops starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true.`